lib/ddtrace/sampler.rb

```diff
@@ -213,13 +213,18 @@ def sample!(span)
       span.sampled = pre_sample?(span) ? @pre_sampler.sample!(span) : true
```

Comment:
⬆️  I'm curious about the comment above.

Is it just a really roundabout way of saying "we don't recommend using pre-sampling?"; otherwise, pre-sampling at 100% means "don't pre-sample".... right?

Reply:
Yes, pre-sampler is all kinds of bad today, mostly because we don't calculate metrics in the library.
I've expanded the comment with the rationale behind it.